models/Recording.js

```diff
@@ -580,6 +580,21 @@ module.exports = function(sequelize, DataTypes) {
 
     const humanSQL = 'NOT "Tags".automatic';
     const AISQL = '"Tags".automatic';
+    if (models.Tag.acceptableTags.includes(tagMode)) {
```

Comment:
Is this right? It doesn't look like `tagMode` would ever be in `acceptableTags`.

Reply:
when you serach on browse for cool that is put into the tagMode. I think we should change this so we sepearte recording Tags and TrackTags in the query

Reply:
Separating the track and recording tags in the query makes a lot of sense.